sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -391,12 +391,12 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * @brief A string value that identifies the portion of the list of blobs to be
```

Comment:
FYI, usually the `@brief` docs is short (single sentence), and then we have `@details` or `@remarks` for the additional details following it.

https://www.doxygen.nl/manual/commands.html

We'd probably want to make such docs changes for the whole SDK as part of a documentation review, including adding docs for parameters, return types, etc., so I don't expect us to change it in this PR, but something to keep in mind going forward.